tasks/post-to-cmr/schemas/config.json

```diff
@@ -31,6 +31,11 @@
         "username": { "type": "string" },
         "password": { "type": "string" }
       }
+    },
+    "skipMetaCheck": {
+      "description": "Adds the option to allow PostToCMR to pass when processing a granule without a metadata file.",
+      "default": false,
```

Comment:
I'm now wondering: do we want to make the default `true` so this isn't a breaking change? But is instead a useful new feature that users can opt-in to?

Reply:
@mennovandiermen pointed out that the intent here is to make `post-to-cmr` fail when it didn't really do what you expect, so this is fine as is